x-pack/plugins/security_solution/public/overview/components/entity_analytics/anomalies/columns.tsx

```diff
@@ -45,13 +45,15 @@ export const useAnomaliesColumns = (
         truncateText: true,
         mobileOptions: { show: true },
         'data-test-subj': 'anomalies-table-column-name',
-        render: (jobName, { count, job }) => {
-          if (count > 0 || (job && isJobStarted(job.jobState, job.datafeedState))) {
-            return jobName;
-          } else {
-            return <MediumShadeText>{jobName}</MediumShadeText>;
-          }
-        },
+        render: (jobName, { count, job }) => (
+          <EuiToolTip content={job?.id ?? jobName}>
```

Comment:
Commenting here for threading again. With regards to installing the same jobs:

> The inclusion of the space as a job id prefix might bring a weird user experience for users with jobs installed in older versions of kibana (without the prefix). **When those users install the same job on 8.8, they will see two versions of it.** That is an uncommon user flow but I updated the UI to display both jobs and added a tooltip to the Entity Analytics page so that users can differentiate them.

How would the user go about installing the same job in `8.8`? Our UI hides the installation behind the enabling of a job, and so it would need to be done from the ML UI, right? When trying that (and including the `space_` prefix), I can't get the jobs to install because it says the datafeed already exists.


<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/220773331-44c2e6cd-f396-48a0-9b4b-57b1a78d4120.png" />
</p>


For references, the setup is as follows:

* Install `Security: Linux` jobs on `main` via Security UI 
* Stop Kibana, check out this PR, verify job is still installed/enabled
* Go to ML UI and select `Security: Linux` module, add `space-name_` prefix, then install fails saying the datafeeds already exist (even though they didn't before trying to create the jobs).

Here's the `jobs_summary` response before (when only global `v3_linux* existed)
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/220772259-096e5b41-cc31-4b79-bc51-fbee0d0984f9.png" />
</p>


And after attempting to install just to the space via the ML UI, where it installed some, but hit a collision with the global one perhaps and stopped there?

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/220771926-8e87b8ee-d94b-464a-b745-f3915d844c7c.png" />
</p>

Reply:
Additionally, it seems like we need some sort of UI to discern global vs space-aware jobs within the Security App other than just a tooltip on the Entity Analytics page, no? Perhaps a 🌐 icon or something simple next to the job name that we can attach a rich tooltip to with details/links to docs.

I guess the bigger question here is are we equally supporting both global and space-aware jobs in our UI's, or just now only supporting space-aware and if you want global jobs you need to install them manually from the ML UI?

Reply:
just chiming in here..

This error is misleading, 
<img width="529" alt="image" src="https://user-images.githubusercontent.com/22172091/220844589-cc24c855-38da-4ccd-bbf2-21905d7f19e8.png">

There are actually two errors here, first saying the job can't be created because the ID is already used.
The second saying the datafeed can't be created because the ID is already used.
I think it's confusing because the sentences are different and it looks like emphasis is being made on the datafeed failure rather than a job and datafeed error.

FYI we still attempt to create the datafeed even though the job failed, just in case we're repairing an install where the datafeed has somehow been deleted.

Reply:
I like the idea of updating the UI to discern global vs space-aware jobs. We currently don't have that information available in the job, but we could call [ /api/ml/saved_objects/jobs_spaces](https://docs.elastic.dev/ml-team/docs/ui/rest-api/ml-api#-get-all-jobs-and-their-spaces) API and get it. We are already calling 3 APIs to list jobs. That would be the fourth 😨

Reply:
About the questions:
> How would the user go about installing the same job in 8.8?

I tried to reproduce the error you saw but couldn't. This user flow doesn't use the code inside this PR because the module installations happen in the ML interface. It successfully installed 2 versions of the same job for me.
<img width="1523" alt="Screenshot 2023-02-23 at 14 42 53" src="https://user-images.githubusercontent.com/1490444/220925096-3f9e14d2-0b9c-4126-9f7f-f76c4510d1a1.png">
<img width="712" alt="Screenshot 2023-02-23 at 14 47 03" src="https://user-images.githubusercontent.com/1490444/220925356-5d149d82-7663-44bf-81c0-103e0114684b.png">

I also was able to install 2 timed the same job by:
* Checkout main and open security
* Install a job on the older version (it installs a package with many jobs)
* Go to the ML interface and delete one job installed in the previous step
* Checkout out this PR
* Open security and reinstall the delete job
* Other jobs in the package will get installed twice.

Reply:
Hrmm, there must be something different between our setups as I'm consistently seeing these jobs fail to install within the ML UI (didn't have a chance to test via Security UI). 

One thing you did different was:
> `Go to the ML interface and delete one job installed in the previous step`

which when I tried it didn't change the outcome for me (same errors as above). I also tested the reverse situation, where the space-aware module/jobs were installed first, then trying to install the global module/jobs and got the same error (also testing each with/without `use dedicated index` just in case you didn't have that selected (Security UI does by default)).